tools/db_crashtest.py

```diff
@@ -1097,6 +1100,12 @@ def whitebox_crash_main(args, unknown_args):
         time.sleep(1)  # time to stabilize after a kill
 
 
+    # If successfully finished or timed out (we currently treat timed out test as passing)
+    # Clean up after ourselves
+    if expected or hit_timeout:
```

Comment:
I think if we reach here at all it must have been a success - there would have been a `sys.exit(1)` already if it had failed

Reply:
I put `expected` check just to be sure in case anything is added in the future to break out of the loop early like we did for timeout, but I can remove it. Please let me know.

Reply:
Keeping it is fine. If we keep it, though, I think `succeeded` would be a better name than `expected`. Also, an initial value of `true` feels more intuitive to me